app/views/personal_access_tokens/_personal_access_tokens_tab.html.erb

```diff
@@ -0,0 +1,61 @@
+<div class="tab-pane" id="personal_access_tokens">
+  <% if flash[:personal_access_token] %>
+    <div class="alert alert-success">
+      <span class="pficon pficon-ok"></span>
+      <strong><%= _('Your New Personal Access Token') %></strong><br />
+      <code id="personal-access-token" style="font-size: 120%;"><%= flash[:personal_access_token] %></code>
+      &nbsp;
+      <a id="personal-access-token-clipboard" data-clipboard-target="#personal-access-token" title="<%= _('Copy to clipboard!') %>">
```

Comment:
nice, i remember we had another place we wanted to use it (maybe rex?) can't think of it it atm. /cc @dLobatog @ares ?